dcos_test_utils/aws.py

```diff
@@ -485,7 +485,7 @@ def get_host_ips(self):
     ),
 }
 
-
+# Update these mappings to expand OS support
```

Comment:
Is this a TODO note? If so, check out "TODO Comment Format" in https://github.com/dcos/dcos/blob/master/docs/styleguides/python.md.